Remove the 'vendored' Cargo feature and its associated dependencies.
This lets us avoid packaging the libftdi1-source-lgpl crate in ChromiumOS.
We already have dev-embedded/libftdi.

diff --git a/Cargo.toml b/Cargo.toml
index 5dc48d1..5d7d93b 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -31,10 +31,6 @@ version = "1.0"
 [dependencies.libc]
 version = "0.2"

-[dependencies.libftdi1-source-lgpl]
-version = "=1.5.0"
-optional = true
-
 [dependencies.libusb1-sys]
 version = ">=0.3.7, <0.7"
 optional = true
@@ -42,16 +38,11 @@ optional = true
 version = "0.59"
 optional = true

-[build-dependencies.cc]
-version = "1.0.52"
-optional = true
-
 [build-dependencies.cfg-if]
 version = "1.0"

 [features]
 default = []
-vendored = ["libftdi1-source-lgpl", "cc"]
 [target."cfg(all(windows, target_env=\"msvc\"))".build-dependencies.vcpkg]
 version = "0.2"
 [target."cfg(not(all(windows, target_env=\"msvc\")))".build-dependencies.pkg-config]
